test/unit/ngsiv2/plugins/multientity-plugin_test.js

```diff
@@ -522,9 +522,10 @@ const iotAgentConfig = {
                     expression: 'z+1'
                 },
                 {
-                    name: 'nonexpectedAtt',
+                    name: 'alsoexpectedAtt',
                     type: 'number',
-                    expression: 'w+1'
+                    expression: 'w+1',
+                    skipValue: 'loquesea'
```

Comment:
Why not also include notnexpectedAtt?
```suggestion
                    expression: 'w+1',
                    skipValue: 'loquesea'
                 },
                {
                    name: 'nonexpectedAtt'
                    type: 'number',
                    expression: 'w+1',
```